yarn.lock

```diff
@@ -8965,7 +8965,7 @@ cypress-real-events@^1.4.0:
   resolved "https://registry.yarnpkg.com/cypress-real-events/-/cypress-real-events-1.4.0.tgz#39575031a4020581e0bbf105d7a306ee57d94f48"
   integrity sha512-1s4BQN1D++vFSuaad0qKsWcoApM5tQqPBFyDJEa6JeCZIsAdgMdGLuKi5QNIdl5KTJix0jxglzFJAThyz3borQ==
 
-cypress@*, cypress@^6.8.0:
+cypress@*:
   version "6.8.0"
```

Comment:
Any reason why this particular block (6.8.0 version of Cypress) still exists in this lock file?

Reply:
Probably because we force it in `package.json`.
We need to resolve to the specific version (6.4.0)

Reply:
Oh, actually no... we apparently removed Cypress from `resolutions`:
https://github.com/metabase/metabase/blob/downgrade-cypress-to-6.4/package.json#L246

Reply:
Good catch, thank you!